### DISCUSSION TITLE
Data not capture on Safari browser

### DISCUSSION THREAD

**Discussion by Emmathem** [react-hook-form/react-hook-form#7141] [Q&A]
Below is the snippet of my code

```
import { useForm } from 'react-hook-form';
 const {
        register,
        handleSubmit,
        formState: { errors },
    } = useForm();
```

```
const ResetPassword = async data => {
    setloading(true);
    setEmailType(data.email);
    const res = await ResetEmailPassword(data);
    setloading(false);
    if (res.success) {
        setShowSuccess(true);
        setloading(true);
        toast.success('Email Reset Link Sent Successfully');
    } else {
        toast.error(res.message);
        setloading(false);
    }
};
```

          <form onSubmit={handleSubmit(ResetPassword)} autoComplete="off">
              <div className="form-group">
                  <input
                      className="input form-control"
                      type="email"
                      placeholder="Email"
                      name="email"
                      {...register('email', {
                          required: true,
                          // pattern: /^[\w-.]+@([\w-]+\.)+[\w-]{2,4}$/g,
                      })}
                  />
                  {errors.email && (
                      <span role="alert" className="error">
                          Email Address is required
                      </span>
                  )}
              </div>
              <Button
                  onClick={handleSubmit(ResetPassword)}
                  type="primary"
                  block
                  loading={loading}
              >
                  Reset Password
              </Button>

          </form>

The above block of code works fine in other browsers aside from Safari. On safari, the value entered in the input is seen as empty at the click of the button.

Who can explain why this? 

Thank you

**Comment by SSylvain1989** [comment]
Hello @Emmathem did you find a solution ?

**Reply by Emmathem** [reply]
@SSylvain1989 Not yet, I haven't gotten a solution

**Reply by petrKavulok** [reply]
any update on this issue?.. @SSylvain1989  @Emmathem

**Comment by leapful** [comment]
@petrKavulok 

Could you please open a new discussion with a codesandbox url to re-produce the issue ? This current discussion is too old and might be obsoleted for the latest "react-hook-form" version.
